docs/faq.md

```diff
@@ -38,3 +38,43 @@ Project Mu makes liberal use of multiple repositories due to the mixture of requ
 repos are split for technical reasons, some for organizational, and some for legal.
 
 For details, see "Repo Philosophy" in [What and Why](WhatAndWhy/overview.md#repo-philosophy).
+
+## Is there any stronger guidance than "Less is More"?
+
+Since Mu is derived from Edk2, the Edk2 coding standards are a good foundation, as is the existing code in the 
+TianoCore repositories. Beyond that, here are a few other musings:
+
+#### Forgotten lessons in firmware
+Firmware, no matter what the flavor of it, all has the same end goal: to securely boot some sort of payload. Features
+have been added to firmware over multiple years, but not all of the features work directly towards the goal of 
+booting a payload.  The Mu repositories attempt to meet the goals of creating a secure environment for firmware.
+
+Some of the additional responsibilities pushed into firmware have become outdated, or can be pared down.  Mu attempts 
+to engage in securing the core requirements for securely booting a system.
+
+
+#### Coding style will change depending on the goal

```

Comment:
The heading title and beginning bullet points led me to believe this was advocating for different people to use different coding styles. I don't think that was the intended message.

But I didn't see that really clarified as I view some of the other topics that follow like "supporting multiple processor architectures" to be "portability". "Code flow" to be "organization". "Minimal entanglement" to be "decoupling". "Industry standard practices" to be "best practices".

Without additional context, I would think of "code style" as the actual style of code as it is written - like variable names, syntax conventions in certain scenarios, etc.

It might help to clearly elaborate what is meant by style in the section intro and then connect the following points to that.